lib/test/cdo/test_slack.rb

```diff
@@ -45,6 +46,11 @@ def test_join_room
     assert Slack.join_room(FAKE_CHANNEL)
   end
 
+  def test_command
+    Slack.expects(:open).returns(stub(read: {'ok' => true}.to_json))
+    assert Slack.command(FAKE_CHANNEL_ID, FAKE_COMMAND)
+  end
```

Comment:
What is this meant to test? 

All it appears to be doing is verifying that `Slack.command(...)` calls `open` (with _any_ parameters), receives a mocked `ok` response, and returns a truthy result. It doesn't verify the parameters on the `open` call. For all we know, it could be opening the wrong url, or passing the wrong parameters.

Should we also test the failure case?